### DISCUSSION TITLE
HMS 1600 4T communication issue

### DISCUSSION THREAD

**Discussion by robsy** [tbnobody/OpenDTU#2413] [Show and tell]
Hi, 

I started some weeks ago with opendtu, and the project is great. 
My setup: 
1x  HMS 1600 4T with 1 panel (425W) on each input, so a bit more than 1600W in total on the flat roof ca. 8m above ground level
1x  ESP32 (ESP32-D0WD-V3) with CMT2300A in the cellar (steel reinforced concrete - Stahlbeton) close o light well (Lichtschacht)

The signal strength is moderate, RSSI ~-82dB
As long as the total Inverter Power is below ca. 250W everything is fine. I poll values every second and the packet loos is below 1%
If the Inverter Power exceeds this level of 250W, the communication stops immediately and completely. Changing Chanel, enhanced transmit power, rebooting ESP32, disconnect from power etc. has no effect.
But putting the ESP and CMT2300A out of the cellar into root at ground floor, communication starts immediately without any failures. RSSI is at this case ~-70dB. Because of the flat autumn sun my max. power was until now 750W, without a single communication failure.
I tested a lot with an USB power bank walking through the house at various sunshine levels and locations and come the conclusion for my HMS1600 4T for RSSI:
- The transmit strength of CMT2300A has (nearly) no influence
- as long as the total output power is below ~250W, RSSI of ~-80dB is enough (by WHY use a 1600W Inverter limited to only 15% ??)
- So the best way: find a place where the RSSI is not lower than ~70dB 

I will add some more comments next summer when the total output will reach higher values.

Robert

P.S. please find attached a output power graph for the 14.11.2024.
The short flat curves between 10:00 and 11:00 was a fault of my internal MQTT server, so nothing to do with this issue from 12:30-13:00 (power > 250W).
![HM_Solar_24-11-14](https://github.com/user-attachments/assets/37411562-ddc6-4abe-bff5-2716ef0f5206)

**Comment by ms49434** [comment]
-82 dBi isn't great. It is always a good idea to install a better (usually omnidirectional) antenna instead of increasing output power. Relocating the unit might not work for everyone.

**Comment by MichaMEG** [comment]
A better antenna, especially for the required frequency, brings a lot. It has significantly improved reception for me.

**Comment by andreas2555** [comment]
Ich habe den E49-900M20S auf eine Platte (PCB) von Ebay aufgelötet und hatte ähnliche seltsame Effekte. Bis ist herausgefunden habe, dass der SMA Antennen-Anschluss gar keine Verbindung zum Modul hatte. Mit einem Adapterkaben IPEX - SMA klappte es dann hervorragend durch mehrere Wände.
